db.py

```diff
@@ -0,0 +1,45 @@
+from json import JSONEncoder, JSONDecoder
+import datetime
+
+
+class Todo:
+    _id = 0
+
+    def __init__(self, task, done=False, date=datetime.datetime.now(), task_id=0):
```

Comment:
import random

def k(r=random.random()):
    print(r)

for i in range(100):
    k()